### PR TITLE
[CHK-562] add opex dashboard configuration file

### PR DESCRIPTION
**Pull request by ciuffagianluca**
#### List of Changes

- added `.identity` folder
- added `.opex` folder
- added github action `create_dashboard.yaml`

#### Motivation and Context

To create dashboard

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .github/workflows/create_dashboard.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        id: login
+        # from https://github.com/Azure/login/commits/master
+        uses: azure/login@92a5484dfaf04ca78a94597f4f19fea633851fa2
+        with:
+          client-id: ${{ secrets.CLIENT_ID }}
+          tenant-id: ${{ secrets.TENANT_ID }}
+          subscription-id: ${{ secrets.SUBSCRIPTION_ID }}
+
+      # this action create a folder named /azure-dashboard
+      - uses: pagopa/opex-dashboard-action@main
```
Comment (pietro-tota) [inline .github/workflows/create_dashboard.yaml:63]: fix this action to checkout a commit

File: .opex/env/uat/config.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+oa3_spec: ./api-spec/api.yaml # If start with http the file would be downloaded from the internet
+name: opex_pagopa-ecommerce-payment-requests-service
+location: West Europe
+timespan: 5m # Default, a number or a timespan https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/scalar-data-types/timespan
+data_source: /subscriptions/b9fc9419-6097-45fe-9f74-ba0641c91912/resourceGroups/pagopa-p-vnet-rg/providers/Microsoft.Network/applicationGateways/pagopa-u-app-gw
```
Comment (pietro-tota) [inline .opex/env/uat/config.yaml:5]:
```suggestion
data_source: /subscriptions/b9fc9419-6097-45fe-9f74-ba0641c91912/resourceGroups/pagopa-u-vnet-rg/providers/Microsoft.Network/applicationGateways/pagopa-u-app-gw
```
This should point to UAT subscription id, right?

File: .identity/02_application_main_auth.tf [hunk 3]
```diff
@@ -0,0 +1,17 @@
+resource "azurerm_role_assignment" "main_terraform_subscription" {
+  scope                = data.azurerm_subscription.current.id
+  role_definition_name = "Reader"
+  principal_id         = azuread_service_principal.main.object_id
+}
+
+resource "azurerm_role_assignment" "main_terraform_storage_account_tfstate_app" {
```
Comment (jacopocarlini) [inline .identity/02_application_main_auth.tf:7]: this is probably not needed

File: .identity/99_outputs.tf [hunk 4]
```diff
@@ -0,0 +1,7 @@
+output "tenant_id" {
```
Comment (jacopocarlini) [inline .identity/99_outputs.tf:1]: add this output:
```
output "client_id" {
 value = azuread_service_principal.main.object_id
}
```

**Review by infantesimone — APPROVED** [review]

**Pull request merged**